src/types.ts

```diff
@@ -0,0 +1,49 @@
+import type { Point, LineString, Polygon } from "geojson";
```

Comment:
@robinlovelace-ate, since you've been having so much fun playing with the JSON schema format, here's a practical example expressed another way. This is ATIP's current geojson schema expressed using TypeScript, and it gets verified throughout the codebase